dotnet/src/IntegrationTests/Connectors/Memory/Milvus/MilvusMemoryStoreTests.cs

```diff
@@ -6,22 +6,19 @@
 using System.Threading.Tasks;
 using Microsoft.SemanticKernel.Connectors.Milvus;
 using Microsoft.SemanticKernel.Memory;
+using Milvus.Client;
 using Xunit;
 
-namespace SemanticKernel.IntegrationTests.Milvus;
+namespace SemanticKernel.IntegrationTests.Connectors.Milvus;
 
-public class MilvusMemoryStoreTests : IAsyncLifetime
+public class MilvusMemoryStoreTests : IClassFixture<MilvusFixture>, IAsyncLifetime
 {
-    private const string MilvusHost = "127.0.0.1";
-    private const int MilvusPort = 19530;
-
-    // If null, all tests will be enabled
-    private const string SkipReason = "Requires Milvus up and running";
```

Comment:
We do run any integration tests that are enabled when we merge the code to main. So we need to reinstate the skip reason and skip all of these tests by default because we don't have the test configuration set-up for Milvus.

Reply:
Isn't that what the test container helps with?

Reply:
@roji  The integration tests are failing in the pipeline see: https://github.com/microsoft/semantic-kernel/actions/runs/8388676975/job/22973369237

Reply:
Thank you @markwallace-microsoft!

If needed, we can make the integration tests opt-in again, as they were before, to unblock things.

Reply:
Expecting this to resolve the integration test failure: https://github.com/microsoft/semantic-kernel/pull/5627